products/absolute-sea-level-heights-baltics-sar-hsu/collection.json

```diff
@@ -88,7 +88,17 @@
   ],
   "osc:status": "ONGOING",
   "osc:region": "Baltics",
-  "osc:type": "Product",
+  "osc:type": "product",
+  "themes": [
+    {
+      "scheme": "OSC:SCHEME:THEMES",
```

Comment:
This still needs a proper URL, see also https://github.com/stac-extensions/osc/issues/4